### PR TITLE
feat : Login Page added

### PR DESCRIPTION
**Pull request by vikrantkode**
# feat : Login Page added with Auth

### CODE REVIEW

File: src/pages/LoginPage/LoginPage.jsx [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+    }
+  }
+
+  const loginDummyClickHandler = async (e) => { 
+    e.preventDefault();
+    try{
+      const resp = await axios.post("/api/auth/login",
+      { email: "adarshbalika@gmail.com",
+        password: "adarshBalika123"})
+      localStorage.setItem("token",resp.data.encodedToken)
```
Comment (krgarima) [inline src/pages/LoginPage/LoginPage.jsx:40]: You are only storing the token. I just want to know are you using it anywhere?

**Review by krgarima — COMMENTED** [review]
Some suggestions: 
1. Change the login button to Log out when user is logged it. Put a log out option too.
2. Put a user friendly error message here which any person can understand. Keep the message simple.
aaa

**Review by dusariudaykumar — APPROVED** [review]
- The UI Looks great
- The code is clean and your file structure is also good.
- Everything is well structured.

**Comment by vikrantkode** [comment]
> Some suggestions:
> 
> 1. Change the login button to Log out when user is logged it. Put a log out option too.
> 2. Put a user friendly error message here which any person can understand. Keep the message simple.
>    aaa

Fix the error message

File: src/pages/LoginPage/LoginPage.jsx (hunk 1 above)
Comment (vikrantkode) [reply src/pages/LoginPage/LoginPage.jsx:40]: No... I am not using it anywhere

**Pull request merged**